packages/react-components/src/components/CameraButton.tsx

```diff
@@ -149,34 +159,54 @@ export const CameraButton = (props: CameraButtonProps): JSX.Element => {
     strings.tooltipDisabledContent = strings.tooltipVideoLoadingContent;
   }
 
+  const cameraOn = props.checked;
+
+  const toggleAnnouncerString = useCallback(
+    (isCameraOn: boolean) => {
+      setAnnouncerString(
+        announcerString === (strings.cameraActionTurnedOffAnnouncement || undefined) || !isCameraOn
+          ? strings.cameraActionTurnedOnAnnouncement
+          : strings.cameraActionTurnedOffAnnouncement
+      );
+    },
+    [announcerString, strings.cameraActionTurnedOffAnnouncement, strings.cameraActionTurnedOnAnnouncement]
+  );
+
   const onToggleClick = useCallback(async () => {
     // Throttle click on camera, need to await onToggleCamera then allow another click
     if (onToggleCamera) {
       setWaitForCamera(true);
       try {
         await onToggleCamera(localVideoViewOptions ?? defaultLocalVideoViewOptions);
+        // allows for the setting of narrator strings triggering the announcer when camera is turned on or off.
+        toggleAnnouncerString(cameraOn ? cameraOn : false);
```

Comment:
if camera was On, button will be checked (confirm this), and was just toggled so we want to announce it is turned off -- should this be `toggleAnnouncerString(!cameraOn)`?

Reply:
so the thought here was to abuse the undefined state of that boolean like a `false` but I will investigate this.